### ISSUE TITLE
Spotify Is Currently Not Running RDP

### ISSUE THREAD

**Issue by Yukatii** [dlrudie/Snip#290]
Hey SNIP Community , sorry if this is a duplicate of previous users with this issue.
Before we start keep in note, RDP is Remote Desktop Connection, A VPS connected through RDP. (VPS Virtual Private Server)
Whenever I have my RDP opened on my main laptop, it works perfectly fine. Then when I attempt to close my RDP it says Spotify Is Currently Not Running. I don't know if this is this is spotify into a different mode such as inactive with music. Any help would be appreciated 

- Thanks.

**Comment by dlrudie** [comment]
See #299
